src/commands/export-sql.ts

```diff
@@ -83,28 +92,37 @@ export const CREATE_EXPORT_JOB_MUTATION = gql`
  * @param {number} envId Environment ID
  * @return {Promise} A promise which resolves to the latest backup and job status
  */
-async function fetchLatestBackupAndJobStatusBase( appId, envId ) {
+async function fetchLatestBackupAndJobStatusBase(
+	appId: number,
+	envId: number
+): Promise< {
+	latestBackup: Backup | undefined;
+	jobs: Job[];
+} > {
 	const api = API();
 
-	const response = await api.query( {
+	const response = await api.query< AppBackupAndJobStatusQuery >( {
 		query: BACKUP_AND_JOB_STATUS_QUERY,
 		variables: { appId, envId },
 		fetchPolicy: 'network-only',
 	} );
 
-	const {
-		data: {
-			app: { environments },
-		},
-	} = response;
+	const environments = response.data.app?.environments;
 
-	const latestBackup = environments[ 0 ].latestBackup;
-	const jobs = environments[ 0 ].jobs;
+	const latestBackup: Backup | undefined = environments?.[ 0 ]?.latestBackup as Backup;
+	const jobs: Job[] = ( environments?.[ 0 ]?.jobs || [] ) as Job[];
```

Comment:
I checked, and defaulting to an empty array is pretty safe. Safer in fact, some functions would crash if `jobs` was not an array.